_config.yml

```diff
@@ -9,10 +9,11 @@ kramdown:
   syntax_highlighter: rouge
 highlights: true
 highlighter: rouge
+timezone: UTC # To match it with Circle CI default settings
 gems:
   - jekyll-redirect-from
   - jekyll-sitemap
-exclude: ['Gemfile*', 'vendor', 'README.md', 'Rakefile']
+exclude: ['Gemfile*', 'vendor', 'README.md', 'Rakefile', 'CNAME']
```

Comment:
If CNAME is excluded from Jekyll, will it still end up in the gh-pages branch when building?